modules/fedimint-ln-client/src/api.rs

```diff
@@ -120,6 +123,14 @@ where
         .await
     }
 
+    async fn get_unspent_incoming_contracts(&self) -> FederationResult<Vec<ContractId>> {
+        self.request_current_consensus(
```

Comment:
From @dpc  https://github.com/fedimint/fedimint/pull/4542#discussion_r1523857020

We need to add an API check here to make sure the server supports this API

Reply:
Added version check